CMakeLists.txt

```
@@ -382,9 +406,15 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
   set(TRITON_PYTORCH_LIBS "${CMAKE_CURRENT_BINARY_DIR}/libtorch.so")
 
   if (${TRITON_PYTORCH_ENABLE_TORCHVISION})
-    set(TRITON_PYTORCH_LIBS
-        ${TRITON_PYTORCH_LIBS}
-        "${CMAKE_CURRENT_BINARY_DIR}/libtorchvision.so.1")
+    if(${RHEL_BUILD})
+      set(TRITON_PYTORCH_LIBS
+          ${TRITON_PYTORCH_LIBS}
+          "${CMAKE_CURRENT_BINARY_DIR}/libtorchvision.so")
+    else()
+      set(TRITON_PYTORCH_LIBS
+          ${TRITON_PYTORCH_LIBS}
+          "${CMAKE_CURRENT_BINARY_DIR}/libtorchvision.so.1")
+    endif()
```

Using [generator-expressions](https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html) is more preferable in my opinion: 
```suggestion
  set(TRITON_PYTORCH_LIBS
          ${TRITON_PYTORCH_LIBS}
     $<IF:$<BOOL:RHEL_BUILD>,${CMAKE_CURRENT_BINARY_DIR}/libtorchvision.so,${CMAKE_CURRENT_BINARY_DIR}/libtorchvision.so.1>)
  )
```
see: https://cmake.org/cmake/help/latest/manual/cmake-generator-expressions.7.html#conditional-expressionsI